src/assets/styles/social.scss

```diff
@@ -45,35 +45,10 @@ $facebook-blue: #3b5998;
         }
       }
     }
-
-    &.facebook {
```

Comment:
why are we doing away with facebook logos? Do we no longer support this feature? Or are they unused?